### PR TITLE
Create getting_started.md

### PR DESCRIPTION
**Pull request by gkorland**
## Summary by CodeRabbit

- **New Features**
	- Introduced a comprehensive guide for setting up and utilizing FalkorDB for modeling a social network as a graph.
	- Included detailed instructions on prerequisites, graph schema, and Cypher query examples for creating and querying data.
	- Provided guidance on connecting to FalkorDB using the Python client.
	- Expanded the wordlist with 119 new entries, enhancing vocabulary support.

- **Bug Fixes**
	- Adjusted formatting in the wordlist by adding a newline at the end of the "yourSourceName" entry.

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+# Getting Started with FalkorDB
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).
```
Comment (swilly22) [inline getting_started.md:18]: Why does Learn More goes to /configuration ?

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).  
+2. **Python Installed**: Ensure you have Python 3.8+ installed.
```
Comment (swilly22) [inline getting_started.md:19]: I think that's obvious, to install the PYTHON client you'll need Python...

File: getting_started.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+   pip install falkordb
+   ```
+   
+---
+
+## Step 1: Model a Social Network as a Graph
+
+Let's create a simple graph for a social network where:  
+- **Nodes** represent `User` and `Post`.
+- **Relationships** represent `FRIENDS_WITH` and `CREATED`.
```
Comment (swilly22) [inline getting_started.md:32]: - **Relationships** connect `Users` with a `FRIENDS_WITH` relation, and `Users` are connected via a `CREATED` relations to `Posts`

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+| FRIENDS_WITH      | User       | User     | `since`      |
+| CREATED           | User       | Post     | `time`       |
+
+---
+
+## Step 2: Load Data into FalkorDB
+
+Here’s how you can model and load the data.
+
+### Cypher Query to Create the Data
```
Comment (swilly22) [inline getting_started.md:52]: the data.

File: getting_started.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+CREATE (bob:User {id: 2, name: "Bob", email: "bob@example.com"})
+CREATE (charlie:User {id: 3, name: "Charlie", email: "charlie@example.com"})
+
+CREATE (post1:Post {id: 101, content: "Hello World!", date: 1701388800})
+CREATE (post2:Post {id: 102, content: "Graph Databases are awesome!", date: 1701475200})
+
+CREATE (alice)-[:FRIENDS_WITH {since: 1640995200}]->(bob)
+CREATE (bob)-[:FRIENDS_WITH {since: 1684108800}]->(charlie)
+CREATE (alice)-[:CREATED {time: 1701388800}]->(post1)
+CREATE (bob)-[:CREATED {time: 1701475200}]->(post2)
```
Comment (swilly22) [inline getting_started.md:65]: date and time as they are right now can be confusing, either use Python to create timestamps from actual dates or consider changing the attribute to something else which doesn't require time / date datatype

File: getting_started.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+---
+
+## Step 2: Load Data into FalkorDB
+
+Here’s how you can model and load the data.
+
+### Cypher Query to Create the Data
+
+```cypher
```
Comment (swilly22) [inline getting_started.md:54]: the queries are repeated in this section and the following, which is a bit of a waste.

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: getting_started.md (hunk 1 above)
Comment (gkorland) [reply getting_started.md:18]: cause it's about how run the Database, what would you suggest?

File: getting_started.md (hunk 2 above)
Comment (gkorland) [reply getting_started.md:19]: right, but you need a specific python version

Comment (gkorland) [reply getting_started.md:19]: It doesn't hurt to mention it

File: getting_started.md (hunk 4 above)
Comment (gkorland) [reply getting_started.md:52]: what is the change you suggest?

File: getting_started.md (hunk 1 above)
Comment (swilly22) [reply getting_started.md:18]: I think learn more should lead to a page with additional explanations and demos not the configuration.

File: getting_started.md (hunk 4 above)
Comment (swilly22) [reply getting_started.md:57]: D -> d

File: getting_started.md (hunk 1 above)
Comment (gkorland) [reply getting_started.md:18]: But we don't have such page,

File: getting_started.md (hunk 5 above)
Comment (gkorland) [reply getting_started.md:70]: how would you do it?

File: getting_started.md (hunk 6 above)
Comment (gkorland) [reply getting_started.md:59]: I know it looks like a repeated section but it's not, the first is giving a pure Cypher example for the data modeling while the second it taking it one step farther and show how to run it in python

File: getting_started.md (hunk 5 above)
Comment (swilly22) [reply getting_started.md:70]: Look up Python date / time to unix timestamp

File: getting_started.md (hunk 1 above)
Comment (swilly22) [reply getting_started.md:18]: then remove the link

File: getting_started.md (hunk 5 above)
Comment (gkorland) [reply getting_started.md:70]: I looked it up, it will bloat the example too much

**Pull request merged**